### PR TITLE
Fix windows scanner crash

### PR DESCRIPTION
**Pull request by vmacari**
https://github.com/hbldh/bleak/issues/1330

### CODE REVIEW

File: bleak/backends/winrt/scanner.py [hunk 1]
```diff
@@ -210,7 +210,7 @@ def _received_handler(
     def _stopped_handler(self, sender, e):
         logger.debug(
             "{0} devices found. Watcher status: {1}.".format(
-                len(self.seen_devices), self.watcher.status
+                len(self.seen_devices), "?" if self.watcher is None else self.watcher.status
```
Comment (dlech) [inline bleak/backends/winrt/scanner.py:213]: How about using `sender` instead?

```suggestion
                len(self.seen_devices), sender.status
```

**Comment by dlech** [comment]
Can you also add a changelog entry?

File: bleak/backends/winrt/scanner.py (hunk 1 above)
Comment (vmacari) [reply bleak/backends/winrt/scanner.py:213]: Sure, that's an option too!

**Review by vmacari — COMMENTED** [review]
Good point, thank you!

**Pull request merged**